lib/shopify_api/event_pipe/worker.ex

```diff
@@ -36,4 +36,7 @@ defmodule ShopifyAPI.EventPipe.Worker do
   defp fetch_token(event) do
     Map.fetch(event, :token)
   end
+
+  def optinally_convert_token(%AuthToken{} = token), do: token
+  def optinally_convert_token(token), do: struct(AuthToken, token)
```

Comment:
So this is making sure that this is a %AuthToken{}, neat. I find the name does not match my expectations, "optionally" to me suggests that maybe it won't return a token rather than maybe it won't need to convert it. Maybe call this `as_auth_token` or `to_auth_token`?

Reply:
that name was bugging me 🤔